apps/web/src/containers/App/App.tsx

```diff
@@ -187,6 +188,11 @@ const AllRoutes = () => {
             component={Markets}
             routeType="Markets"
           />
+          <AppLayoutRoute
+            path="/analytics"
```

Comment:
Can you route this to `/analytics/transactions`? In the future we will have more views under this